docs/docs/glossary/infrastructure-as-code.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Infrastructure as Code
+disableTableOfContents: true
+---
+
+Learn what <q>Infrastructure as Code</q> means, and how you can use can use code to standardize and automate your Gatsby site.
+
+## What is <q>Infrastructure As Code?</q>
+
+_Infrastructure As Code_ is the practice of managing your development, testing, and production environments using configuration files or scripts. Provisioning and configuring environments individually can introduce errors or inconsistencies. You may, for example, find yourself running different versions of Node on your laptop and your production servers. Infrastructure As Code mimimizes this kind of drift and lets you automate the process of provisioning environments.
+
+Your configuration file describes what resources your project requires. If you're building an API, for example, you might create a configuration file that says, "Please install Node 12.16.2, npm 6.14.4, Express 4.17.1, and PostgreSQL 12.2 for Ubuntu Linux." For a Gatsby project, your configuration file may add plugins and themes.
+
+Configuration files, like other code files, are text. That means you can use version control software to store them and track changes to the environment. In short, infrastructure as code:
+
+- Creates consistent environments.
+- Saves time that would otherwise be spent setting up environments.
+- Reduces the risk of errors caused by mismatched environments.
+- Makes it easy to automate tasks.
```

Comment:
Rather than "easy" maybe a more polished version of "sets you up to automate tasks"?

Reply:
Would "simplifies the automation of tasks" or "simplifies task automation" work?

Reply:
Went with "Lays the groundwork for..."